WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift

```diff
@@ -0,0 +1,119 @@
+import UIKit
+import GoogleSignIn
+
+class SignupEpilogueViewController: NUXViewController {
+
+    // MARK: - Properties
+
+    var googleUser: GIDGoogleUser?
+    private var buttonViewController: NUXButtonViewController?
+    private var updatedDisplayName: String?
+    private var updatedPassword: String?
+
+    // MARK: - View
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
+    // MARK: - Navigation
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+
+        if let vc = segue.destination as? NUXButtonViewController {
+            buttonViewController = vc
+            buttonViewController?.delegate = self
+            buttonViewController?.setButtonTitles(primary: NSLocalizedString("Continue", comment: "Button text on site creation epilogue page to proceed to My Sites."))
+        }
+
+        if let vc = segue.destination as? SignupEpilogueTableViewController {
+            vc.loginFields = loginFields
+            vc.googleUser = googleUser
```

Comment:
I think `googleUser` should be added to `LoginFields`. Seems like we need it everywhere `LoginFields` is, and it will take fewer lines to do so.